apps/assisted_tagging/enums.py

```diff
@@ -9,17 +9,26 @@
     DraftEntry,
     AssistedTaggingPrediction,
 )
+from lead.models import Lead
 
 DraftEntryPredictionStatusEnum = convert_enum_to_graphene_enum(
     DraftEntry.PredictionStatus, name='DraftEntryPredictionStatusEnum')
 AssistedTaggingPredictionDataTypeEnum = convert_enum_to_graphene_enum(
     AssistedTaggingPrediction.DataType, name='AssistedTaggingPredictionDataTypeEnum')
+DraftEntryTypeEnum = convert_enum_to_graphene_enum(
+    DraftEntry.Type, name="DraftEntryTypeEnum"
+)
+AutoEntryExtractionTypeEnum = convert_enum_to_graphene_enum(
+    Lead.AutoExtractionStatus, name="AutoEntryExtractionTypeEnum"
+)
```

Comment:
As enums have global naming, Let's add Model name as prefix.
```suggestion
LeadAutoEntryExtractionTypeEnum = convert_enum_to_graphene_enum(
    Lead.AutoExtractionStatus, name="LeadAutoEntryExtractionTypeEnum"
)
```